static/js/widgetize.js

```diff
@@ -315,6 +315,153 @@ widgets.poll = (function () {
     return cls;
 }());
 
+var trivia_data_holder = function () {
+    var self = {};
+
+    self.has_answered = {};
+    self.correctly_answered = false;
+    self.msg = "";
+
+    self.trivia_question = {};
+    var correct_answer;
+
+    self.set_trivia_ques = function (result) {
+        var question = result.results[0].question;
+
+        correct_answer = result.results[0].correct_answer;
+
+        var options = result.results[0].incorrect_answers;
+        options.push(correct_answer);
+        self.trivia_question  =  {
+            question : question,
+            options : _.shuffle(options),
+            correct_answer :  correct_answer,
+        };
+    };
+
+    self.promise = $.ajax({
+            url: "https://opentdb.com/api.php?amount=1&type=multiple",
+        });
+
+    self.handle = {
+        option_click: {
+            outbound: function (option) {
+                var event = {
+                    type: 'option_click',
+                    option: option,
+                    trivia_question : self.trivia_question,
+                    correct_answer : correct_answer,
+                };
+                return event;
+            },
+
+            inbound: function (sender_id, data) {
+                self.trivia_question = data.trivia_question;
+                correct_answer = $("<div/>").html(data.correct_answer).text();
```

Comment:
See if you can figure a way to not use jquery in the data class.  We really only need the correct answer here, so we could populate that we create the instance of this data_holder class, I think.

(The goal is to separate the data piece from the UI piece as much as possible.)